lib/components/artist/articles/index.js

```diff
@@ -16,7 +16,10 @@ class Articles extends React.Component {
     return (
       <View>
         <SerifText style={styles.heading}>Featured Articles</SerifText>
-        <ScrollView horizontal={true} style={{ overflow: 'visible', marginBottom:40 }}>
+        <ScrollView horizontal={true}
+                    showsHorizontalScrollIndicator={false}
+                    scrollsToTop={false}
```

Comment:
so... setting it to false in the react code will allow the native code to work as it should, scrolling to the top on a statusbar tap?